### PR TITLE
fix css distortion in grants sidebar

### LINKED ISSUE #7391: Dropdown arrows in categories section overlap to next line
**Describe the bug**

Hello. I noticed the categories section on the sidebar has some drop-down arrows that are overlapping where the text is longer in length. 

**To Reproduce**

When the [grants page](https://gitcoin.co/grants/) is visited

**Expected behavior**

Dropdown arrows in the categories section to stay on the side of text.

**Screenshots**

Screenshot 2020-09-15 at 08 49 37

**Desktop (please complete the following information):**

- Browser: Chrome
- Browser Version: 85.0.4183.102

### PR DESCRIPTION
**Pull request by chibie**
##### Description

Fix distortion in grants sidebar categories

before:

image

after:

image

##### Fixes

Fixes #7391

### CODE REVIEW

**Comment by aamustapha** [comment]
This still falls on 2 lines.
image20

I think a good fix might have to make the root view flex and setting a minimum width on the sidebar_search.

**Comment by chibie** [comment]
> This still falls on 2 lines.
> image20
> 
> I think a good fix might have to make the root view flex and setting a minimum width on the sidebar_search.

 "$50k - Space Race 2 (Filecoin)"
is too long a name for category. there should be a max character length for category name considering there's also a superscript

max length of 20 characters works well. as shown in this sample using 9999 for matic

**Comment by chibie** [comment]
@aamustapha 

the sidebar_search already uses up the full width of col-3 for 3-9 column set

"$50k - Space Race 2 (Filecoin)"
is too long a name for category. there should be a max character length for category name considering there's also a superscript

a max length of 20 characters works well. as shown in this sample using 9999 superscript for matic

image

**Comment by aamustapha** [comment]
Well, in this case I'd recommend either, increasing the width of the sidebar a little or, reducing the size of the font a little. I just feel it'd be neat if we're able to wrap everything on one line.

You could also decide to go rogue and use `overflow: visible` 🤡

**Comment by chibie** [comment]
i guess the filecoin category has been renamed

image

**Review by aamustapha-gitcoin — APPROVED** [review]
LGTM

**Review by octavioamu — APPROVED** [review]

File: app/assets/v2/css/sidebar_search.css [hunk 1]
```diff
@@ -106,6 +107,7 @@
     margin-left: 10px;
     margin-right: 10px;
     padding: 20px 20px 10px;
+    width: 100%;
```
Comment (octavioamu) [inline app/assets/v2/css/sidebar_search.css:110]: this file is shared on tribes and other part of the site I think should be better instead just use w-100 class on the html to be safe is not going to break nothing on the other pages

**Review by octavioamu — CHANGES_REQUESTED** [review]
Left a comment just for caution

File: app/assets/v2/css/sidebar_search.css (hunk 1 above)
Comment (danlipert) [reply app/assets/v2/css/sidebar_search.css:110]: Yeah this is probably a good idea @chibie - we often find when we change css thats used on multiple pages some visual errors someplace else weeks later... 😆

**Review by danlipert — CHANGES_REQUESTED** [review]
see Octavio's comment

File: app/assets/v2/css/sidebar_search.css (hunk 1 above)
Comment (chibie) [reply app/assets/v2/css/sidebar_search.css:110]: oh right! i'll fix this in a bit

Comment (chibie) [reply app/assets/v2/css/sidebar_search.css]: @octavioamu done!

**Review by danlipert — APPROVED** [review]

**Pull request merged**